src/test/emulators/functionsEmulator.spec.ts

```diff
@@ -54,6 +54,50 @@ describe("FunctionsEmulator-Hub", () => {
       });
   }).timeout(TIMEOUT_LONG);
 
+  it("should route requests to /:project_id/:region/:trigger_id/ to HTTPS Function", async () => {
+    UseFunctions(() => {
+      require("firebase-admin").initializeApp();
+      return {
+        function_id: require("firebase-functions").https.onRequest(
+          (req: express.Request, res: express.Response) => {
+            res.json({ hello: "world" });
```

Comment:
In these two tests instead of hello world let's return:
```
res.json({ path: req.path })
```

And then assert that the function gets the `path` value we expect.  So `/` in this case and ``/a/b` in the next.